app/templates/main/userProfile.html

```diff
@@ -492,6 +492,7 @@ <h5 class="modal-title">Add Notes</h5>
                   {% if g.current_user.isCeltsAdmin or g.current_user.isCeltsStudentStaff %}
                     <option value='2'>Admin/Student Staff</option>
                   {% endif %}
+                  <!-- Ala, we mightt need to add something here -->
```

Comment:
Ala, do we know if we need something here yet?

Reply:
thanks for pointing that out!